pkg/startup/v4/data/master/etc/origin/node/node-config.yaml

```diff
@@ -33,6 +33,16 @@ kubeletArguments:
   - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
   max-pods:
   - "50"
+  image-gc-high-threshold: 
+  - "80"
+  image-gc-low-threshold: 
+  - "60"
+  maximum-dead-containers:
+  - '50'
+  maximum-dead-containers-per-container:
+  - '2'
+  minimum-container-ttl-duration:
+  - 5m
```

Comment:
Don't change v4 and v5. Only `v6`. It's not si critical issue we would need to do v4 release for this.

Reply:
fixing